### DISCUSSION TITLE
Will frigate work on my thin client (AMD G-Series GX-212JC))?

### DISCUSSION THREAD

**Discussion by Tobster77** [blakeblackshear/frigate#11771] [Ask A Question] [answered]
### What is your question:

I use a thin client type [HP 520](https://support.hp.com/de-de/document/c04387430) with 8GB RAM and an AMD G-Series GX-212JC dual core processor ([with AVX](https://www.cpu-world.com/CPUs/Bulldozer/AMD-G-Series%20GX-212JJ.html)), running Home Assistant with about 5% CPU load and 15% RAM usage. Will it be capable for frigate, at least for motion-triggered recording of one or two cameras? Will there maybe be enough CPU capacity for human being detection, or is this not realistic?

Thanks for your estimates...

**Comment by NickM-27** [comment]
CPU based detection is not recommended in any setup, frigate supports a lot of different hardware accelerated object detection options. For just motion based detection, I think that would work alright for ~4 cameras

**Comment by Tobster77** [comment]
Thank you. As this is an older SoC with embedded graphics, the GPU cababilities will be very limited, see [here](https://www.cpu-world.com/CPUs/Puma/AMD-G-Series%20GX-212JC.html). I assume the only hardware acceleration possible would be buying a Coral USB extension?

**Reply by NickM-27** [reply]
there are two types of hardware acceleration and both are highly recommended. Hardware accelerated video decoding and object detection. A coral helps on the object detection side

**Comment by Tobster77** [comment]
Okay. As the CPU was introduced in 2014, I would hope it is compatible with the [Radeon 2000 driver](https://docs.frigate.video/configuration/hardware_acceleration/#amdati-gpus-radeon-hd-2000-and-newer-gpus-via-libva-mesa-driver) (LIBVA_DRIVER_NAME=radeonsi). Is this worth a try?

**Comment by Tobster77** [comment]
Sadly, I run frigate as a Home Assistant add-on. So no ```radeonsi``` driver, right?

**Reply by NickM-27** [reply, marked as answer]
correct
